packages/@okta/vuepress-theme-prose/layouts/AuthorizationLayout.vue

```diff
@@ -19,9 +19,14 @@
 </template>
 
 <script>
+import pendo from '../util/pendo'
+
 export default {
   components: {
     Footer: () => import('../components/Footer.redesign')
+  },
+  mounted: async function() {
+    await pendo;
```

Comment:
This async/await bit here is useless if there's nothing to be executed after it, same comment for all 3 files